uxarray/grid/grid.py

```diff
@@ -242,7 +244,9 @@ def __init__(
     cross_section = UncachedAccessor(GridCrossSectionAccessor)
 
     @classmethod
-    def from_dataset(cls, dataset, use_dual: Optional[bool] = False, **kwargs):
+    def from_dataset(
+        cls, dataset, use_dual: Optional[bool] = False, is_subset=False, **kwargs
```

Comment:
Can we exclusive `is_subset` from the parameters and access it through the `kwargs`?

Can you pass in `is_subset=kwargs.get('is_subset', False)`  into the `cls()` constructor below?